administrator/components/com_patchtester/config.xml

```diff
@@ -21,29 +37,55 @@
 			label="COM_PATCHTESTER_FIELD_REPO_LABEL"
 			description="COM_PATCHTESTER_FIELD_REPO_DESC"
 			default="joomla-cms"
+			id="repo"
+			showon="repository:custom"
 		/>
 
+	</fieldset>
+
+	<fieldset
+		name="component"
```

Comment:
solved.

BTW do you know any way to solve the autocomplete

The only way i know is to change the field type from password to something else.